dm/syncer/schema.go

```diff
@@ -65,6 +66,24 @@ func (s *Syncer) OperateSchema(ctx context.Context, req *pb.OperateWorkerSchemaR
 		// in other words, we can not get the schema if any DDL/DML has been replicated, or set a schema previously.
 		return s.schemaTracker.GetCreateTable(ctx, sourceTable)
 	case pb.SchemaOp_SetSchema:
+		// from source or target need get schema
+		if req.FromSource {
+			schema, err := dbconn.GetTableCreateSQL(s.tctx.WithContext(ctx), s.fromConn, sourceTable.String())
```

Comment:
found `sourceTable.String()` is not escaping backquotes... hope we won't meet that scenario.